internal/locate/region_request.go

```diff
@@ -1064,6 +1065,24 @@ func (s *RegionRequestSender) SendReqCtx(
 			}
 		}
 
+		//resp = nil
```

Comment:
Is this temporary?

Reply:
It's updated in [my repo](https://github.com/you06/client-go/tree/reset-stale-read-6.5), but the display in PR is inconsistent.